pywhat/Data/regex.json

```diff
@@ -463,6 +463,19 @@
           "Credentials"
        ]
     },
+    {
+      "Name": "American Social Security Number",
+      "Regex": "([0-9][0-9][0-9])(?<!666|000|9[0-9][0-9])(=|\\+|_|\\#|:|;|\\.|-|\\s){0,3}([1-9][1-9]|0[1-9]|[1-9]0)(=|\\+|_|\\#|:|;|\\.|-|\\s){0,3}([0-9][0-9][0-9][0-9])(?<!0000)(=|\\+|_|\\#|:|;|\\.|-|\\s){0,3}?",
+      "plural_name": false,
+      "Description": null,

```

Comment:
We had description before. Can you use it here?

Reply:
I didn't understand what use it had so I removed it. Would you like me to put it back?

Reply:
Yes

Reply:
Done